doc/source/ray-air/examples/huggingface_text_classification.ipynb

```diff
@@ -2152,9 +2152,9 @@
    },
    "outputs": [],
    "source": [
-    "from ray.train.huggingface import LegacyTransformersCheckpoint\n",
+    "from ray.train.huggingface import TransformersCheckpoint\n",
     "\n",
-    "checkpoint = LegacyTransformersCheckpoint.from_checkpoint(result.checkpoint)\n",
+    "checkpoint = TransformersCheckpoint.from_checkpoint(result.checkpoint)\n",
```

Comment:
This needs to be `TransformersCheckpoint(path=result.checkpoint.path, filesystem=result.checkpoint.filesystem)`